draft-ietf-quic-http.md

```diff
@@ -1396,22 +1396,25 @@ servers, the initial value of each client setting is the default value.
 For clients using a 1-RTT QUIC connection, the initial value of each server
 setting is the default value. When a 0-RTT QUIC connection is being used, the
 initial value of each server setting is the value used in the previous session.
-Clients MUST store the settings the server provided in the session being resumed
-and MUST comply with stored settings until the current server settings are
-received.  A client can use these initial values to send requests before the
+Clients SHOULD store the settings the server provided in the session being
+resumed and MUST comply with stored settings until the current server settings
```

Comment:
I might be confused, but how can you have `MUST comply with stored` if storing is a `SHOULD`?

Reply:
Yeah, this complicates the text somewhat.  If the intent is to have clients comply with either defaults or whatever they remember, then maybe:

Clients SHOULD store the settings the server provided in the connection where resumption information was provided.  A client MUST comply with either stored settings - or default values if no values are stored - when attempting 0-RTT.  Once a server has provided new settings, clients MUST comply with those values.